routes/web.php

```diff
@@ -43,9 +43,7 @@
 
         Route::resource('orders.items', 'OrderItemsController', ['only' => ['update']]);
 
-        Route::get('/', function () {
-            return ujs_redirect(route('admin.store.orders.index'));
-        });
+        Route::get('/', '\App\Http\Controllers\RedirectController')->name('redirect.admin.store.orders.index');
```

Comment:
might as well make a function which does this

```php
function route_redirect($path, $target)
{
    // using `:` because not used anywhere else for route name
    return Route::get($path, '\App\Http\Controllers\RedirectController')->name("redirect:{$target}");
}
```